content/docs/pipelines/sdk/pipelines-metrics.md

```diff
@@ -67,8 +67,8 @@ ML - TFX - Taxi Tip Prediction Model Trainer** from the Kubeflow Pipelines UI.
 For help getting started with the UI, follow the 
 [Kubeflow Pipelines quickstart](/docs/pipelines/pipelines-quickstart/).
 
-The sample code is available in the [Kubeflow Pipelines samples 
-repo](https://github.com/kubeflow/pipelines/tree/master/samples/tfx). The
+The sample code is available in the [Kubeflow Pipelines samples
```

Comment:
@gaoning777  I'm thinking that the tfx-oss sample may not provide the same metrics as we've illustrated here. Is that right? If so, I recommend we leave the screenshots as they are in this doc at this point, for illustrative purposes, but remove lines 64-79.

Reply:
@sarahmaddox should I also remove the next list items? Does it have value within the context of the pipeline?

Reply:
@butuzov I recommend removing lines 64-79. I think those lines include the list items that you're mentioning?

Reply:
Yep, that ones. Well... done.